src/other/Table.js

```diff
@@ -290,11 +305,14 @@
             })
         ;
         rows
-            .attr("class", function (d) {
+            .classed("selected", function (d) {
                 if (context._selectionBag.isSelected(context._createSelectionObject(d))) {
-                    return "selected";
+                    return true;
```

Comment:
Yes, of course... I had more code in these functions earlier that I removed, did not think to condense it